cmd/swapd/main.go

```diff
@@ -148,7 +140,7 @@ var (
 			},
 			&cli.StringFlag{
 				Name:  flagEthereumEndpoint,
-				Usage: "Ethereum client endpoint",
+				Usage: "Ethereum websockets client endpoint",
```

Comment:
With this PR we now fail quickly (instead of slow and painfully), if the user passes an http/https endpoint.

Reply:
why was it failing before?

Reply:
@dimalinux I've been running the node with http endpoints always and never had any issues?

Reply:
I reverted it back and created a ticket to remove the dependency on websocket URLs:
https://github.com/AthanorLabs/atomic-swap/issues/356